solidity/random-beacon/README.adoc

```diff
@@ -177,103 +166,122 @@ When the hard timeout is reached, anyone can notify about this fact by calling
 `RandomBeacon.reportRelayEntryTimeout()` function and receive a notifier
 reward. The group which failed to submit a relay entry is terminated, group
```

Comment:
Maybe link 'notifier reward' (or earlier 'anyone can notify') to Punishment section in Transaction Incentives?

Reply:
1fadd0bedacc1e8332f6ef3c24e1f6db4acc3322